### DISCUSSION TITLE
Using other people's bank account

### DISCUSSION THREAD

**Discussion by liang3zy22** [community/community#56716] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello everyone,

I am liang3zy22 from China. I have met an issue now. My bank account country is not in the 30 countries list. Can I use other people's bank account which is in the 30 countries list? The bank account name is not my name. Not sure if I can use other people's bank account.

Thanks.

liang3zy22

**Comment by DHayes10** [comment, marked as answer]
Hi @liang3zy22, 

Thank you for reaching out. Please click [here](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account) for more information on submitting your bank information. 

Best, 
GitHub Sponsors

**Comment by liang3zy22** [comment]
Thanks, @DHayes10 . This sentence "If you choose to receive payouts to a bank account, your region of residence and the region of your bank account must match" answered my question.
